src/style.css

```diff
@@ -0,0 +1,130 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap");
+
+body {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__container {
+  display: flex;
+  padding: 0 3px;
+  margin-bottom: 40px;
+}
+
+.card__imac-img {
+  width: 160px;
+  height: 134px;
+}
+
+.card__content {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  color: #060b35;
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__feedback {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 24px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+}
+
+.stars--color_yellow :nth-child(-n + 4) {

```

Comment:
You should complete this requirement:
![image](https://user-images.githubusercontent.com/25587299/219368148-080afce9-07f4-4c7c-ab09-0bde2569f744.png)

So you should have all possible modifiers: `.stars--1`, `.stars--2`, ..., `.stars-5`.